### DISCUSSION TITLE
EBUS Data Wolf heater professional level / "Fachmann Ebene"

### DISCUSSION THREAD

**Discussion by MiniOh** [john30/ebusd#918] [General]
(german below)

Hello,

I have a question about the professional level (Fachmann Ebene) of Wolf heaters.
I have now managed to query the data via EBUS and pass it on via MQTT.

It is now the case that "only" the values are read out which are accessible without a professional level (Fachmann Ebene).
Further data is only output via EBUS when I log on to the heating system myself as a "professional" with the code 1111.

The same also works if I register as a professional via the Wolf Portal Smart Set.
Further data is then also communicated via the EBUS.
However, only for about 15 minutes, until the Expert level is "closed" again.

Is there a possibility that the specialist data can be accessed permanently via EBUS?

Thank you in advance.

==================================================

Hallo, 

ich habe eine Frage zu der Fachmann Ebene von Wolf Heizungen.
Ich habe es nun geschafft, die Daten via EBUS abzufragen und diese via MQTT weiter zu geben.

Nun ist es so, dass "nur" die Werte ausgelesen werden, welche ohne Fachmann Ebene zugreifbar sind.
Erst wenn ich mich an der Heizung selbst als "Fachmann" mit dem Code 1111 anmelde, werden auch weitere Daten via EBUS ausgegeben.

Das gleiche funktioniert auch wenn ich mich über das Wolf Portal Smart Set als Fachman anmelde.
Auch dann werden weitere Daten über den EBUS kommuniziert.
Allerdings nur für ca. 15 Minuten, bis die Fachmann Ebene wieder "beendet" wird.

Gibt es eine Möglichkeit, dass die Fachmann Daten dauerhaft über EBUS abgreifbar sind?

Danke schon mal vorab.

**Comment by mwildbolz** [comment]
Maybe you can poll also the data out of "Fachmannebene" via ebusd by setting a poll priority inside the configuration files - r1 to r9, not just r

BR

**Comment by MiniOh** [comment]
Thx for the fast answer.

You mean the first "r" in each row?

**r**;u,SM2,TK1_24h_MAX,,,76,5022,a57809,,,SIN,10,Â°C,Temperatur_Kollektor_1_24h_max,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

**Reply by mwildbolz** [reply]
right, see also https://github.com/john30/ebusd/wiki/4.1.-Message-definition#message-definition

**Comment by MiniOh** [comment]
Thanks.
Unfortunately, this does not change anything about reading out the specialist data.

**Comment by MiniOh** [comment]
I have to retract my previous statement.
It works with "r1".

Thank you again

**Reply by mwildbolz** [reply]
Glad to hear.
Just one hint from my side:
I had the problem, that I had too much values included in the polling cycles and polled them too often. The result was, that i got strange values e.g. for outsideTemperature (-40°C). There are also some threads regarding this or similar topics.
I solved it with reducing polling cycles and using mainly values sent regularly by the system itself (e.g. Status01 messages, etc.)

**Reply by andrelung** [reply]
can you share your csv files?

**Comment by MiniOh** [comment]
OK, thanks for the tip.
Another question, do you have a Wolf or a Vaillant heater?

**Reply by mwildbolz** [reply]
Vaillant

**Comment by MiniOh** [comment]
A pity.
I'm still looking for good config files.
